### ISSUE TITLE
Bug: Generated c# client methods return voids

### ISSUE THREAD

**Issue by fairking** [RicoSuter/NSwag#3996]
NSwag v13.15.10.0 (NJsonSchema v10.6.10.0 (Newtonsoft.Json v11.0.0.0))
Windows 10
.Net6
Visual Studio 2022 Preview 6.0

I was trying to do that via VS (Add Service Reference -> OpenAPI) but it wasn't working. So I simply run the command bellow in powershell (`C:\Users\Admin\.nuget\packages\nswag.msbuild\13.15.10\tools\Win\NSwag.exe`).

The command:
```
NSwag.exe openapi2csclient /className:API /namespace:AccessPay /input:{path}swagger.json /output:{path}swaggerClient.cs
```

The swagger scheme: https://eazycollectservices.github.io/EazyCollectAPIv3/swagger.json

The above example generates not compilable client code:
``` c#
if (status_ == 200)
{
    return default(void); // Compile error
}
```

![image](https://user-images.githubusercontent.com/13495631/166894360-b3e8db41-3b19-4466-b558-96bf5c7a002e.png)

**Comment by fairking** [comment]
Could be related to https://github.com/RicoSuter/NSwag/issues/3912

**Comment by crackalak** [comment]
@fairking yes I think this is related. This happens because your swagger json doesn't define the content for the response types, it only seems to have examples (there should be a content property under the 200) e.g:

```json
"responses": {
          "200": {
            "description": "OK",
            "examples": {
              "application/json": {
                "schema": {
                  "example": {
                    "Message": "Callback URL cleared."
                  },
                  "type": "string"
                }
              },
              "application/xml": "<GeneralAcknowledgement\n  xmlns:xsd=\"http://www.w3.org/2001/XMLSchema\"\n  xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\">\n    <Message>Callback URL cleared.</Message>\n</GeneralAcknowledgement>"
            }
          }
        },
```

See spec for more info https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md#responses-object-example

**Comment by fairking** [comment]
@crackalak Thank you for the response. So the issue is related to the missing api docs. Closing the issue.

**Comment by tmatous** [comment]
The OpenAPI specs do not require a content definition. And even if it did, NSwag should not generate invalid C# code. This issue should be reopened.

**Comment by pavlexander** [comment]
agreed. There is no excuse for returning the broken code. If it comes down to errors in the specification (which is not the case according to @tmatous in this particular scenario), then the **nswag** should at least show some warnings or even an error during the code generation process.. 

Just stumbled upon the same issue..
